### ISSUE TITLE
Add aria-label to sidenav

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/vets-design-system-documentation#1999]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

The sidenav currently doesn't have a unique label so it cannot be differentiated clearly from other navigation landmarks on the page.
![image](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/106678594/774354d8-2884-4968-8156-12c95b49e3b7)

## What I expected to happen

The sidenav component should have an `aria-label` on the `<nav>` element so that when it is used on a page it is able to be differentiated from other navigation landmarks by assistive technology.

## Reproducing
- Formation version:
- Device: (e.g. iPhone 8, Macbook)
- Browser: (e.g. Firefox, IE 11)

Steps to reproduce:

1.
2.
3.
4.

## Urgency
How urgent is this request? Please select the approriate option below and/or provide details
- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [x] Other

**Comment by micahchiang** [comment]
@laflannery  DST doesn't own the sidenav, this should be handed off to the sitewide team.

**Comment by laflannery** [comment]
@micahchiang We already did this on Sitewide, I made this thinking something would need to be updated within the DS because there was was [a page here](https://design.va.gov/components/sidenav). But I guess if there is nothing for the DS to do you all can close this?
